app/models/address.rb

```diff
@@ -0,0 +1,8 @@
+class Address < ApplicationRecord
+  belongs_to :user
+  validates :street, :city, :state, :zip, presence: true
+
+  def complete_address
+    street + ", " + city + ", " + state + ", " + zip
```

Comment:
Like the `complete_address` method but could we remove the last comma? I'm a fan of the address being displayed like this: `1331 17th St ll100, Denver, CO 80202`

Reply:
That's a great point! it'll be an easy fix!